spring-data-jpa/src/main/java/org/springframework/data/jpa/repository/query/KeysetScrollSpecification.java

```diff
@@ -63,7 +60,7 @@ public KeysetScrollSpecification(KeysetScrollPosition position, Sort sort, JpaEn
 	 */
 	public static Sort createSort(KeysetScrollPosition position, Sort sort, JpaEntityInformation<?, ?> entity) {
 
-		KeysetScrollDirector director = KeysetScrollDirector.of(position.getDirection());
+		KeysetScrollDelegate director = KeysetScrollDelegate.of(position.getDirection());
 
```

Comment:
If `KeysetScrollDirector` is no more, then the variable name probably needs to change as well.